modules/tensor_mechanics/doc/content/source/utils/RankTwoScalarTools.md

```diff
@@ -0,0 +1,249 @@
+# Rank Two Scalar Tools
+
+
+## Description
+
+This is a utility class that provides a set of functions to compute scalar
+quantities for stress or strain tensors. This class is not directly invoked in
+the input file, but is called by several other classes that are, including:
+
+## Axial Stress
+
+The scalar type `AxialStress` calculates the scalar value of a Rank-2 tensor,
```

Comment:
The scalar type does not compute anything. These enum values are only relevant for the calling class. In this file you should describe the actual functions in this file.